v0/pkg/prometheus/gen/updater.go

```diff
@@ -88,18 +87,12 @@ func updateMapped(d *collector.DynamicMap, rlr *label.RecursiveReflector, fv ref
 			delete(keysToDelete, rk)
 			continue
 		}
-		var keyString string
```

Comment:
I think it would be good to have a test for this.

Reply:
@AndreasBergmeier6176 
Changes are covered in 'v0/pkg/prometheus/gen/collector_test.go' if this is okay for you